### DISCUSSION TITLE
Visibility of context menu item based on extension of selected file

### DISCUSSION THREAD

**Discussion by k5tQp2ET** [moudey/Shell#546] [Q&A]
I added new item to context menu, and I want to make it visible only for specific type of file .e.g. .txt, .dds, .ini,

**Comment by RubicBG** [comment]
`find='.txt|.dds|.ini'` - the simplest way, works 99% of the situations
or
` where=(sel.file.ext=='.txt' or sel.file.ext=='.dds')`
or
`where=str.contains('.txt|.dds|.ini|', sel.file.ext+'|')`

**Comment by k5tQp2ET** [comment]
Thank you.
